backend/gen_timestamp.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Do we need to submit this file?

Reply:
I think it's ok, it's used to generate random data, will be removed as I work on the given 2 datasets